include/ProgramRepresentation/DisjointPVAliasSets.h

```diff
@@ -26,6 +26,9 @@ class DisjointPVAliasSets {
     // returns a copy of every disjointed set
     std::list<PVAliasSet> getSets() const;
 
+    // returns the contained alias set that has matching references to the list of Program variables
```

Comment:
I _think_ this means that the returned set matches _all_ of the elements of `vars` (as opposed to the set that includes _any one_ of the elements in the list), but it would be a good idea to clarify that in this comment.